Source/WebCore/page/EventHandler.h

```diff
@@ -136,6 +137,10 @@ class EventHandler {
     explicit EventHandler(LocalFrame&);
     ~EventHandler();
 
+    // For CheckedPtr / CheckedRef use.
+    void incrementPtrCount() const;
```

Comment:
I think we should just make EventHandler inherit from `CanMakeCheckedPtr` unless this is absolutely necessary for correctness, in which case the reason should be stated in the commit message.

Reply:
I think this goes against @darinadler 's recommendation on my earlier PR?

The EventHandler is completely owned by the Frame via a UniqueRef so we don't need to increase the size of EventHandler unnecessarily. I also made EventHandler::m_frame be a CheckedRef<Frame> so that the lifetime safety is bi-directional.

I don't mind either way. I merely went this way to be consistent with my earlier PR where I think Darin suggested this (the review comments were not 100% clear to me so I may have misunderstood as well).